portfolio/src/main/webapp/script.js

```diff
@@ -33,5 +33,9 @@ function addRandomFact() {
 
   // Add it to the page.
   const factContainer = document.getElementById('fact-container');
-  factContainer.innerText = fact;
+  var para = document.createElement("p");
+  var node = document.createTextNode(fact);
```

Comment:
Similarly to my previous comment that became obsolete, you could instead select the original `p` element on the page and simply update its `innerHtml` value each time. You could prepopulate the `fact-container` with an empty `p` tag, if needed.